js/main.js

```diff
@@ -1,44 +1,58 @@
-function copyMatsList() { // copies matsList to clipboard
+function selectOption() {
+    let select = document.getElementById("dropdown-select");
+    let selected = select.value;
+    recipesViewer(selected);

```

Comment:
could just do `receipesViewer(selected.value)`

Though to be honest, this isn't the most necessary function split. It would be fine to keep this all as one function